src/main.rs

```diff
@@ -54,15 +54,15 @@ async fn main() -> io::Result<()> {
     init_logging(&log_args);
 
     if let Err(err) = run(command).await.as_ref() {
-        tracing::error!(error = ?err, "network tunnel failed.");
+        tracing::error!(error = %err, "network tunnel failed.");
```

Comment:
% signifies the `Display` implementation of the error, rather than its `Debug` implementation, so the error looks more human-readable rather than exposing the underlying error struct